### DISCUSSION TITLE
Feature request: ability to directly load a specific chat from CLI

### DISCUSSION THREAD

**Discussion by Andrew15-5** [ayn2op/discordo#794] [Ideas]
Although #195 is how I currently navigate between chats (GUI), if I can alias a bunch of them then it would be quicker to type the alias and load the specific chat directly, rather than open the app, wait for it to load, then find the right chat manually or via #195 (and then wait for the chat to load). Instead, it would be `discordo --open-chat <full or trimmed URL>` and then you wait for the app to load everything once and you are good to go.

**Comment by ayn2op** [comment]
Related: #195
